pkg/meta/route.go

```diff
@@ -0,0 +1,18 @@
+package meta
+
+import (
+	"inspr.dev/inspr/pkg/utils"
+)
+
+// Route is an Inspr component that represents a route,
+// which is a http request address related to the endpoints
```

Comment:
Replace second line: Endpoints is a list of allowed endpoints on regarding the route